Scripts/Tracking/Velocity/ApplyVelocity.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace VRTK.Core.Tracking.Velocity
+{
+    using UnityEngine;
+
+    /// <summary>
+    /// Applies the velocity data from the given <see cref="VelocityTracker"/> to the given <see cref="Rigidbody"/>.
+    /// </summary>
+    public class ApplyVelocity : MonoBehaviour
+    {
+        /// <summary>
+        /// The <see cref="Rigidbody"/> to apply the velocity data to.
+        /// </summary>
+        [Tooltip("The Rigidbody to apply the velocity data to.")]
+        public Rigidbody applyTo;
+        /// <summary>
+        /// The <see cref="VelocityTracker "/> to receive the velocity data from.
+        /// </summary>
+        [Tooltip("The VelocityTracker to receive the velocity data from.")]
+        public VelocityTracker velocityData;

```

Comment:
This is not necessarily `data`, instead I suggest `source`. We don't need to talk about `velocity` in the member name here as it's implied by the component type we're in and the member's type.